packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -210,6 +210,17 @@ export class MetriportMedicalApi {
     return resp.data as PatientDTO;
   }
 
+  /**
+   * Searches for a patient.
+   *
+   * @return The patient if found.
+   */
+  async searchPatient(data: PatientSearch): Promise<PatientDTO> {
```

Comment:
if we are adding an endpoint, lets also add it to the fern definition so customers using the python sdk get access to it? If you need a pointer or two on how fern works just hit me up :)

Reply:
Done.